app/views/claims/existing_session.html.erb

```diff
@@ -1,38 +1,30 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <div class="govuk-panel govuk-panel--interruption">
-      <%= form_with url: start_new_path(journey: current_journey_routing_name), method: :post do |form| %>
-        <fieldset class="govuk-fieldset" aria-describedby="start_new_claim">
+      <%= form_with url: start_new_path(journey: current_journey_routing_name), builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+        <% radio_options = [
+              OpenStruct.new(id: true, name: "Yes, start claim #{journey_description(current_journey_routing_name)} and lose my progress on my first claim"),
+              OpenStruct.new(id: false, name: "No, finish the claim I have in progress"),
+            ] %>
```

Comment:
i think leaving this inline for now is sufficient. we may want to switch to a view component at a later date but starting to get diminishing returns for effort spent